docs/installation/cloud.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "Installation on a cloud platform"
+description = "Installation instructions for Docker on Amazon EC2."
+keywords = ["cloud, docker, machine, documentation,  installation"]
+[menu.main]
+parent = "smn_cloud"
++++
+<![end-metadata]-->
+
+# Cloud providers
+
+You can install Docker on any cloud platform that runs an operating system (OS) that Docker supports. This includes many flavors and versions of Linux, along with Mac and Windows.
+
+__REVIEWERS NOTE:__ The plan is to remove all the detailed info on each cloud provider that's currently under the "cloud install" topic (https://docs.docker.com/machine/drivers/). Are the following very general steps enough to replace that with?
```

Comment:
@thaJeztah Reviewers notes are fairly common in the writer world. I don't have a problem with writer's doing them in preference over inline notes.  They are an advantage to put them in text --- rare is the writer that forgets to take them out.   If it becomes a problem, we can Automate the fix.  

Another compromise would be to put them in comment blocks.  That way they are still here but unlikely to go out.  @londoncalling comment blocks don't have the nice bold tho.